frontend/public/components/masthead-toolbar.jsx

```diff
@@ -22,6 +23,25 @@ const UpdatesAvailableButton = ({obj, onClick}) => {
     : null;
 };
 
+const onServiceMesh = (url, name, e) => {
+  e.preventDefault();
+  window.open(`${url}?namespace=${name}`, '_blank').opener = null;
+};
+
+const ServiceMeshItem = ({namespace}) => {
+  if (namespace.loaded) {
+    const ns = namespace.data;
+    const serviceMeshUrl = _.get(ns, 'metadata.annotations["openshift.servicemesh/url"]');
+    const name = ns.metadata.name;
+    return serviceMeshUrl ?
+      <DropdownItem key="servicemesh" onClick={(e) => onServiceMesh(serviceMeshUrl, name, e)}>
+        OpenShift ServiceMesh
```

Comment:
```suggestion
        Service Mesh
```

Reply:
Serena said it should be called 'OpenShift ServiceMesh', second comment in https://jira.coreos.com/browse/CONSOLE-1212

Reply:
OK, we should follow up on that. Everywhere I've seen it, it's "Service Mesh" (two words). For instance:

https://istio.io/docs/concepts/what-is-istio/

Reply:
Ok, cc'ing @serenamarie125

Reply:
Changed to 'Service Mesh'